examples/src/main/java/com/fingerprint/example/FunctionalTests.java

```diff
@@ -44,7 +44,7 @@ public static void main(String... args) {
 
         // Search events
         try {
-            final Object events = api.searchEvents(2, null, "bad", null, null, null, null, null, null);
+            final Object events = api.searchEvents(2, new FingerprintApi.SearchEventsFilter().setBot("bad"));
```

Comment:
Outside the scope of the PR, I am just curious, why are events typed as `Object` and not `SearchEventsResponse` ?

Reply:
I don't remember reasons for it, but fixed.